recipes/antismash/meta.yaml

```diff
@@ -49,8 +49,16 @@ about:
   license: AGPL-3.0
   license_family: AGPL
   license_file: LICENSE.txt
-  summary: "antiSMASH allows the rapid genome-wide identification, annotation and analysis of secondary metabolite biosynthesis gene clusters."
+  summary: antiSMASH - the antibiotics and Secondary Metabolite Analysis SHell
+  description: |
+        antiSMASH allows the rapid genome-wide identification,
+        annotation and analysis of secondary metabolite biosynthesis gene clusters.
```

Comment:
Looks like the `home` directory is showing the web server page, so if you want, you can use/add:
```
home: https://docs.antismash.secondarymetabolites.org/intro/
dev_url: https://github.com/antismash/antismash
doc_url: https://docs.antismash.secondarymetabolites.org
```
instead.

Reply:
Good.